# parameters for tsdf_converter.cpp
tsdf_converter_jump: 1
voxel_size: 0.02
downsampling_rate: 100

min_x: 0.5 # 0.5
max_x: 1.8 # 1.8
min_y: 0.5 # 0.5
max_y: 1.5 # 1.5
min_z: 0.75 # 0.75
max_z: 1.50 # 1.45

# cluster_extraction.cpp
cluster_extraction_leaf_size: 0.01 # 0.01
cluster_tolerance: 0.01 # 0.01
min_cluster_size: 100 # 100
max_cluster_size: 25000 # 25000
cluster_extraction_distance_threshold: 0.02 # 0.02

# cluster_projection.cpp
table_tolerance: 0.05

# plane_recognition.cpp
plane_recognition_leaf_size: 0.01 # 0.02
plane_recognition_distance_threshold: 0.04 # 0.04
publish_plane_marker: true

# kinfu
using_head_camera: true
table_cutoff_above: 0.03

# tsdf zero-crossing threshold
tsdf_min_distance: 0 # 0
tsdf_max_distance: .4 # .4
